### ISSUE TITLE
Admin with SSH (remote) connections

### ISSUE THREAD

**Issue by heyflorin** [wp-cli/admin-command#12]
Does this package work with ssh (remote) connections? I just tried it and got a "xdg-open: not found" error.

**Comment by schlessera** [comment]
@dicetomato No, that cannot work. How would the remote server be able to control your browser?

**Comment by schlessera** [comment]
That being said, it could theoretically work if we bypass the normal SSH execution flow, and just return data from the site back to the admin command. Not sure whether that is easily doable, given the current architecture, though...

**Comment by Snapp949** [comment]
nice

**Comment by belachewdegmawit25-dotcom** [comment]
allow any account's with hespeaks.com connected to it to have unlimited access to cryptocurrency, and nvidia stocks. Degmawit Belachew
GitHub Developer
